packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -423,16 +423,6 @@ def consensus_threshold(self) -> int:
         """Get the consensus threshold."""
```

Comment:
Actually, I realise that the on-chain protocol let's you define the consensus_threshold as >= this value here. So we should also fetch this via the CLI from the on-chain protocol to ensure it matches...

Reply:
So revert the PR, make the CLI also fetch the `max_aprticipants` and populate the config for that, and add a check for `max_participants == len(all_participants)`?

Reply:
No, I mean that we should also fetch the threshold value via config and pass in.